packages/common/src/grpc-service.js

```diff
@@ -161,6 +161,11 @@ function GrpcService(config, options) {
 
   this.grpcMetadata = new grpc.Metadata();
 
+  this.grpcMetadata.add('User-Agent', [
```

Comment:
We expect the user agent to be set with the channel argument "grpc.primary_user_agent". So, you would construct a client like this:

```js
client = new Client(address, credentials, {'grpc.primary_user_agent': user_agent_string});
```

Note that this will add your user agent string to the existing grpc user agent string.